src/ui/classic/classicui.h

```diff
@@ -102,7 +102,7 @@ FCITX_CONFIGURATION(
         useInputMethodLanguageToDisplayText{
             this,
             "UseInputMethodLangaugeToDisplayText",
```

Comment:
should we also change the typo in "UseInputMethodLangaugeToDisplayText"? change to "UseInputMethodLanguageToDisplayText"?